docs/source/commandline/commandline-all.rst

```diff
@@ -305,6 +557,34 @@ Example:
     *pattern* or system. Note that this is simply a copy/paste of the .XML file
```

Comment:
This is kind of misleading.  It combines all the software lists that it finds for the specified system(s), and it will lose comments, unrecognised elements/attributes, formatting, etc.  I think this command was designed for front-ends, but it’s not that really that useful because of performance.  A front-end is better off mapping machines/slot cards to software lists upfront using `-listxml` output.

Reply:
I don't know who wrote that originally, but on a second glance that's really bad phrasing. I'll try to come up with something better for you to look at shortly.